server/svix-server/src/queue/mod.rs

```diff
@@ -79,6 +135,7 @@ impl TaskQueueConsumer {
     }
 }
 
+#[derive(Clone)]
```

Comment:
I'm always weary about seeing clones, because well, they clone. Why is this needed all of a sudden?

Reply:
Good find, this was added during debugging but ultimately not needed, shouldn't be in this PR.